userguide/getting-started/includes/tomcat.adoc

```diff
@@ -177,7 +179,38 @@ In addition to the Kill Bill database, you will also need to configure the Kaui
 
 . Run the Kaui DDL https://github.com/killbill/killbill-admin-ui/blob/master/db/ddl.sql[here].
 
-. You should have a database called *kaui* (Or whatever name you have specified in step 1) with the following tables:
+==== PostgreSQL Configuration
+
+In order to configure PostgreSQL, you need to follow the steps given below (These steps can be executed either via the command line *psql* tool or via *pgAdmin*):
+
+.  Create a database corresponding to kaui. You can run the following command:
+[source,sql]
+ CREATE DATABASE kaui WITH OWNER = killbilluser;
+
+.  Connect to the kaui database created above. In psql, you can run the following:
+[source,sql]
+  \connect kaui;
+
+.  Execute https://github.com/killbill/killbill/blob/master/util/src/main/resources/org/killbill/billing/util/ddl-postgresql.sql[this] schema extension DDL.
+
+.  Run the Kaui DDL https://github.com/killbill/killbill-admin-ui/blob/master/db/ddl.sql[here].
+
+.  Grant privileges to the schema using the following command:
+[source,sql]
+  GRANT ALL ON SCHEMA public TO killbilluser;
+
```

Comment:
These steps look different from the Kill Bill Postgres steps [here](https://docs.killbill.io/latest/development.html#_postgresql_configuration). Could you verify both sets of instructions once?

Reply:
Yes, initially I granted only on the tables but there was errors in the logs regarding sequences. So I granted on all to the user.